CONTRIBUTING.md

```diff
@@ -97,3 +97,16 @@ To build the package:
 ```
 grunt build
 ```
+
+## Using aXe with TypeScript
+
+The TypeScript definition file for axe-core can be found in [typings/axe-core](./typings/axe-core).
+
+After running `npm install` to install Typescript, you can run TS definition tests (with the optional `noImplicitAny` flag): 
+
+```
+ntsc --noImplicitAny typings/axe-core/axe-core-tests.ts
```

Comment:
It should be `tsc`! I think I got confused with the many rewrites and library deprecations.